### DISCUSSION TITLE
Add a drop-down menu for NFTs being offered

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#1084] [Ideas]
Currently (1.6.1 rc2) when offering an NFT, you have to enter the NFT's ID. It would be nice to have a dropdown list of NFTs that you own, from which you can then select a thumbnail of the NFT you would like to offer. This would save the extra step of copying and pasting the ID. While it is still possible to select "Create Offer" directly from an NFT, if you want to offer more than one NFT, you will need to know the IDs of each subsequent NFT.

**Comment by paninaro** [comment]
It's in the works...

**Comment by BrandtH22** [comment]
This has been completed and included in the GUI in a previous version of chia
